apps/snitch_core/test/data/model/zone/state_zone_test.exs

```diff
@@ -84,6 +88,69 @@ defmodule Snitch.Data.Model.StateZoneTest do
     end
   end
 
+  describe "get/1" do
+    test "zone with id", %{zone: zone} do
+      {:ok, new_zone} = StateZone.get(zone.id)
+      assert new_zone.id == zone.id
+    end
+  end
+
+  describe "get_all/0" do
+    @tag state_zone_count: 3,
+         country_zone_count: 3
+    test "zones  successfully", %{zones: zone} do
+      state_zones = StateZone.get_all()
+      sz = Enum.drop_while(state_zones, fn x -> x.zone_type == "S" end)
```

Comment:
I have used it to access all the struct records inside the list(state_zones). I have used it because only the zone_type field was common to all the records present inside the list(state_zones).